### PR TITLE
Currencies: Write tests

### PR DESCRIPTION
**Pull request by oreqizer**

### CODE REVIEW

File: src/currency/queries/__tests__/Currencies.spec.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { graphql, RestApiMock } from '../../../common/services/TestingTools';
+import currenciesDataset from '../../datasets/currencies.json';
+
+beforeEach(() => {
+  RestApiMock.onGet(
+    'https://nitro-hankey.skypicker.com/currencies',
+  ).replyWithData(currenciesDataset);
+});
+
+describe('#Currencies', () => {
```
Comment (michalsanger) [inline src/currency/queries/__tests__/Currencies.spec.js]: Why the `#`?

File: src/currency/queries/__tests__/Currency.spec.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { graphql, RestApiMock } from '../../../common/services/TestingTools';
+import currenciesDataset from '../../datasets/currencies.json';
+
+beforeEach(() => {
+  RestApiMock.onGet(
+    'https://nitro-hankey.skypicker.com/currencies',
+  ).replyWithData(currenciesDataset);
+});
+
+describe('#Currencies', () => {
```
Comment (michalsanger) [inline src/currency/queries/__tests__/Currency.spec.js]: `Currency`

File: src/currency/queries/__tests__/__snapshots__/Currency.spec.js.snap [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "round": 0,
+      "uncertainFormat": false,
+    },
+  },
+}
+`;
+
+exports[`#Currencies getting a non-existent currency 1`] = `
+Object {
+  "data": Object {
```
Comment (michalsanger) [inline src/currency/queries/__tests__/__snapshots__/Currency.spec.js.snap]: Some error message should be added. Try this query:
```
{
  location(id: "not_exist") {
    locationId
    name
    slug
  }
}
```

File: src/currency/queries/__tests__/Currencies.spec.js (hunk 1 above)
Comment (oreqizer) [reply src/currency/queries/__tests__/Currencies.spec.js]: looks nice in the output

Comment (michalsanger) [reply src/currency/queries/__tests__/Currencies.spec.js]: But inconsistent :-(

**Comment by oreqizer** [comment]
@michalsanger fixed all things

File: src/currency/queries/__tests__/Currencies.spec.js (hunk 1 above)
Comment (oreqizer) [reply src/currency/queries/__tests__/Currencies.spec.js]: k

Comment (mrtnzlml) [reply src/currency/queries/__tests__/Currencies.spec.js]: > But inconsistent :-(

The same for `*.spec.js` for tests and so on. It's not the first inconsistency... :)

Comment (michalsanger) [reply src/currency/queries/__tests__/Currencies.spec.js]: @mrtnzlml right, that can be fixed easily. Could you create an issue with other inconsistencies you know about? 

Looks like #63 is really needed

Comment (mrtnzlml) [reply src/currency/queries/__tests__/Currencies.spec.js]: I don't think it makes sense without eslint rules or something like that. I miss a lot of things even with my OCD... 🙂 I knew about this even in the first PR but we allow it so it's not necessarily wrong. The same thing with the tests style. We usually use the BDD syntax `it` but not here. And again - it's not wrong, or is it?

I will be working on #63 today.

Comment (michalsanger) [reply src/currency/queries/__tests__/Currencies.spec.js]: ESLint could be part of the PR. There is nothing like good/bad with this, it's only about agreement and write it up into `CONTRIBUTING.md`

**Pull request merged**